packages/webdriverio/src/commands/browser/$.ts

```diff
@@ -57,7 +57,7 @@ export default async function $ (this: WebdriverIO.BrowserObject, selector: stri
      * convert protocol result into WebdriverIO element
      * e.g. when element was fetched with `getActiveElement`
      */
-    if (selector && typeof (selector as ElementReference)[ELEMENT_KEY] === 'string') {
+    if (typeof selector !== 'string' && typeof (selector as ElementReference)[ELEMENT_KEY] === 'string') {
```

Comment:
Note that `typeof selector !== 'string'` only applies if `selector` is the primitive data type `string`, if you also want to check for all objects of type `String`, you would need to do:
```
!(typeof selector === 'string' || selector instanceof String)
```
(but at this point I would probably extract it to a `util` function)

Reply:
it doesn't matter in this case